back-end/schemas/schemas.sql

```diff
@@ -1,6 +1,61 @@
 -- write the database name that you use here
 USE db_name;
 
+-- create database project_5;-- 
+-- use project_5 
+CREATE table productCategory(
+categoryId int  NOT NULL ,
```

Comment:
Use the correct convention for naming columns, it should be `category_id` not `categoryId` 
You will have to fix it in all of the tables, also make sure to go to your code and use the new names over there too
In VS Code press `ctrl + shift + f ` to search all the files for a specific keyword then you can change all of them